tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -1308,6 +1317,9 @@ bool BaseGpuFusionStrategy::isFusible(Operation* op) {
   if (isa<lmhlo::ReduceOp>(op) &&
       (!isRank2RowReduction(op) && !isRank2ColReduction(op)))
     return false;
+
+  if (isa<lmhlo::TransposeOp>(op) && isRank2Transpose(op)) return false;
```

Comment:
better to add knob to protect this part of code and disable this by default.

Reply:
done